packages/admin/cms-admin/src/userPermissions/hooks/currentUser.tsx

```diff
@@ -42,11 +38,20 @@ export const CurrentUserProvider: React.FC = ({ children }) => {
 
     if (!data) return <Loading behavior="fillPageHeight" />;
 
-    return <CurrentUserContext.Provider value={Object.assign(new CurrentUser(), data.currentUser)}>{children}</CurrentUserContext.Provider>;
+    const context: CurrentUserContext = {
+        currentUser: data.currentUser,
+        isAllowed: (user: CurrentUserInterface, permission?: string) => {
+            if (user.email === undefined) return false;
+            if (!permission) return true;
```

Comment:
what is the usecase for undefined permission?

Reply:
There is none, I changed it: 1ce11b84dea531f59e4062c8fe524009ab627351